src/status_im/ui/screens/accounts/events.cljs

```diff
@@ -32,9 +32,9 @@
 
 (handlers/register-handler-fx
  :account-created
- [re-frame/trim-v (re-frame/inject-cofx ::get-signing-phrase) (re-frame/inject-cofx ::get-status)]
+ [re-frame/trim-v (re-frame/inject-cofx :get-signing-phrase) (re-frame/inject-cofx :get-status)]
```

Comment:
we want to stop using the trim-v coeffect

Reply:
what do we want to replace it with?